pkg/infra/game/agent/strategy.go

```diff
@@ -52,3 +54,13 @@ type Loot interface {
 type HPPool interface {
 	DonateToHpPool(baseAgent BaseAgent) uint
 }
+
+type Trade interface {
+	// HandleTradeNegotiation given a map of trade negotiations, respond to one of them or start a new trade negotiation
+	HandleTradeNegotiation(
+		agent BaseAgent,
```

Comment:
I think reducing this signature could be a good thing to add by adding a struct for the parameters, I understand that the rest of the API isn't as structured but since this is a breaking change, might as well adopt some good standards now 🤣